components/RandomCover/index.js

```diff
@@ -20,29 +26,32 @@ export default function RandomCover({ artworks }) {
       Math.floor(Math.random() * filteredRandomArtworks.length)
     ];
 
+  setTimeout(hideLoadingAnimation, 5000);
```

Comment:
This works well. But I'd recommend making use of `useEffect` here and putting the `setTimeout` within the callback function. And by using **the dependeny array** you have control over when exactly the animation should disappear.

Reply:
Thank you :) I will try this!